doc/lua_api.txt

```diff
@@ -5119,12 +5119,16 @@ This is basically a reference to a C++ `ServerActiveObject`
   `set_look_vertical`.
 * `set_look_yaw(radians)`: sets look yaw - Deprecated. Use
   `set_look_horizontal`.
-* `get_breath()`: returns players breath
-* `set_breath(value)`: sets players breath
+* `get_breath()`: returns player's breath
+* `set_breath(value)`: sets player's breath
     * values:
         * `0`: player is drowning
         * max: bubbles bar is not shown
-        * See [Object properties] for more information
+        * See Object Properties for more information
```

Comment:
Capitalization should match header (lowercase properties)

Reply:
Oops, that happened while fixing a merge conflict. Good catch.